paymentsheet/src/main/java/com/stripe/android/paymentsheet/viewmodels/BaseSheetViewModel.kt

```diff
@@ -289,6 +294,8 @@ internal abstract class BaseSheetViewModel(
                         previouslyShownForm = null
                         previouslyInteractedForm = null
                     }
+                    is PaymentSheetScreen.Form -> {
```

Comment:
should we call reportFormShown in this branch instead of for PaymentSheetScreen.VerticalMode?

Reply:
It's a good question. I don't know! Let's plan on adding a task to handle the analytics in a follow up. I just had to add this to make the compiler happy.